fakebook/src/environments/environment.prod.ts

```diff
@@ -1,4 +1,4 @@
 export const environment = {
   production: true,
-  baseUrl: 'someUrl' //replace with backend production url
+  baseUrl: 'https://fakebook.revaturelabs.com/api'
```

Comment:
this is not the right baseurl. there is no `/api` prefix in the current code on the backend.

Reply:
fixed